### PR TITLE
PR #3: Frontend Teams subfolder feature dropdown when creating a new draft

### PR DESCRIPTION
**Pull request by yashcode00**
🚀 **PR 3: Summary:**

The following are the things done in this PR:

1. Added the new subfolder classification namely Teams (Inside the BU), made frontend UI for selecting it via the same dropdown mechanism used for selecting the product (BU), while creating a new draft using ember.js and there very. own Helous Design System. 
2. Made a dummy fetch API which fetched from some random static predefined json for the time being.

> [cherry-picked from [d0c375d](https://github.com/razorpay/hermes/pull/5/commits/d0c375d8c607bc0b0f149ef65d45f3a637a5603a)]

Thank you! 😊

### CODE REVIEW

File: web/app/components/header/nav.hbs [hunk 1]
```diff
@@ -43,6 +43,14 @@
         @icon="file-plus"
         class="create-draft-button"
       />
```
Comment (soujash-mandal) [inline web/app/components/header/nav.hbs:45]: why two same buttons ? what is the difference between them

File: web/app/components/new/doc-form.ts [hunk 2]
```diff
@@ -211,6 +236,7 @@ export default class NewDocFormComponent extends Component<NewDocFormComponentSi
             contributors: this.getEmails(this.contributors),
             docType: this.args.docType,
             product: this.productArea,
+            // team: this.teamArea,
```
Comment (soujash-mandal) [inline web/app/components/new/doc-form.ts:239]: how are you sending the teamarea info to backend ?

File: web/app/components/new/doc-form.ts [hunk 3]
```diff
@@ -211,6 +236,7 @@ export default class NewDocFormComponent extends Component<NewDocFormComponentSi
             contributors: this.getEmails(this.contributors),
             docType: this.args.docType,
             product: this.productArea,
```
Comment (soujash-mandal) [inline web/app/components/new/doc-form.ts:238]: product means BU here right?

**Review by ashisht99x — APPROVED** [review]

File: web/app/components/header/nav.hbs (hunk 1 above)
Comment (yashcode00) [reply web/app/components/header/nav.hbs:45]: Yes they are duplicate thanks for noticing I will change it!

Comment (yashcode00) [reply web/app/components/header/nav.hbs:45]: Yes they are duplicate thanks for noticing I will change it!

**Pull request merged**